libs/coin-bitcoin/src/api/index.ts

```diff
@@ -55,10 +55,10 @@ export const getFeeItems = async (currency: CryptoCurrency): Promise<FeeItems> =
     const blockCount = parseInt(key, 10);
     if (blockCount === defaultBlockCount) defaultFeePerByte = feePerByte;
 
-    if (!Number.isNaN(blockCount) && !feePerByte.isNaN() && blockCount in speeds) {
+    if (!Number.isNaN(blockCount) && !feePerByte.isNaN() && speeds.has(blockCount)) {
```

Comment:
Makes sense to use Map to me if key is supposed to be a number, we can do this update. Else it's a "sidegrade"

Just beware as speeds.has(blockCount) when blockCount is a number will return true but won't with a string compared to blockCount in speeds

![image](https://github.com/LedgerHQ/ledger-live/assets/19901996/e5b7e232-afdd-4171-87c6-2b3ed4b66fc8)

Reply:
> Just beware as speeds.has(blockCount) when blockCount is a number will return true but won't with a string compared to blockCount in speeds

It is expected to returns false, as `"1"` and `1` is not the same thing. On line 55, `blockCount` is declared as an int.

Keeping an object instead of a map in this case, lead to a type error on line 61. To resolve it, it will imply more code to change

Reply:
ok, using a map as you did is fine with me :+1: